### PR TITLE
refactor: angular integration guide

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Update angular integration guide

### CODE REVIEW

File: docs/quick-starts/framework/angular/_get-user-information.mdx [hunk 1]
```diff
@@ -13,10 +15,86 @@ The `buildAngularAuthConfig()` utility will enable `autoUserInfo` and `renewUser
 To learn more about configuring the `angular-auth-oidc-client` library, see the [official documentation](https://angular-auth-oidc-client.com/).
 :::
 
-Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC userinfo endpoint. Both of the "scope" and the "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.
+### Display user information
 
-In short, when you request a scope, you will get the corresponding claims in the user information. For example, if you request the `email` scope, you will get the `email` and `email_verified` data of the user.
+The `OidcSecurityService` provides a convenient way to subscribe to the authentication state:
```
Comment (wangsijie) [inline docs/quick-starts/framework/angular/_get-user-information.mdx:20]: nit: including user information

**Review by wangsijie — APPROVED** [review]

File: docs/quick-starts/framework/angular/_get-user-information.mdx (hunk 1 above)
Comment (charIeszhao) [reply docs/quick-starts/framework/angular/_get-user-information.mdx]: updated

File: docs/quick-starts/framework/angular/_get-user-information.mdx [hunk 2]
```diff
@@ -13,10 +15,86 @@ The `buildAngularAuthConfig()` utility will enable `autoUserInfo` and `renewUser
...
+        // highlight-end
+      }),
+    }),
+    // ...other providers
+  ],
+};
+```
+
+Then you can access the additional claims in the return value of `userData`.
+
```
Comment (gao-sun) [inline docs/quick-starts/framework/angular/_get-user-information.mdx:97]: section: user info that requires network request?

Comment (charIeszhao) [reply docs/quick-starts/framework/angular/_get-user-information.mdx:100]: Updated

image
